prow/plugins/label/label_test.go

```diff
@@ -390,18 +523,37 @@ func TestLabel(t *testing.T) {
 	}
 
 	for _, tc := range testcases {
-		fakeClient, ice := getFakeRepo(tc.body, tc.commenter, tc.repoLabels, tc.issueLabels)
-		if err := handle(fakeClient, logrus.WithField("plugin", pluginName), ice); err != nil {
-			t.Errorf("For case %s, didn't expect error from label test: %v", tc.name, err)
-			continue
-		}
+		for i := 0; i < 3; i++ {
```

Comment:
Noted. that's what i think before too. mind if i use reflect package for it (no deepequal)?